content/architecture/vault.md

```diff
@@ -18,13 +18,223 @@ toc: true
+```
+
+  1.  The program prompts you to choose your Google Cloud Platform
+project from the available list.
+
+  2.  The program prompts you to update your existing managed servers
+to use the displayed list of Cloud DNS nameservers. Copy the
+list for use in the next steps.
+
+  3.  From the Google Cloud Platform
+[Zones](https://console.cloud.google.com/net-services/dns/zones)
+page, change the *Resource Record Type* to `NS`) and use the default
+values for your domain for for *TTL* (`5`) and *TTL Unit*
+(`minutes`).
+
+  4.  Add the first nameserver to the *Name server* field
+
+  5.  Click `Add item` and add any subsequent nameservers.
+
+  6.  Click `Create`.
+
+  7.  Configure Jenkins X for the new domain names:
+
+    - Edit the `jx-requirements.yaml` file and update the `domain`
+field (in `ingress`) to your domain name, for example
+`cluster1.acmecorp.example`
+
+    - In the *tls* setting, enable TLS with `enabled: true`
+
+    - The resulting `jx-requirements.yaml` entries for these settings
+should look similar to the following:
+
+```yaml
+gitops: true
+ingress:
+  domain: cluster1.acmecorp.example
+  externalDNS: true
+  namespaceSubDomain: -jx.
+  tls:
+email: certifiable@acmecorp.example
+enabled: true
+production: true
+secretStorage: vault
+```
+
+Run `jx boot` for the changes to take effect in your
+environment.
+
+## Creating a Vault
+
+A vault is created by default using [jx boot](/getting-started/boot/) to create your cluster, unless you specified during the cluster configuration not to create the vault. In this case, you can create one post-installation
+with the `jx create` command-line interface:
+
+jx create vault
+
+1.  The program will ask you the name you want for your vault (for
+example, `acmevault`)
+
+2.  The program will ask you for your Google Cloud Zone of choice. Refer
+to [Regions and
+Zones](https://cloud.google.com/compute/docs/regions-zones/) in the
+Google Cloud documentation for more information. In this example,
+`us-east1-c` is chosen for proximity to Acme Headquarters.
+
+3.  If you have a storage bucket account configured from creating a
+cluster with `jx boot`, then the `jx create vault` command will scan
+your installation for Vault-related storage buckets and, if found,
+prompt you to approve deleting and recreating the Vault from
+scratch.
+
+4.  The program will ask you the *Expose type* for your vault in order
+to create rules and routes for cluster load balancer and other
+services. Default is `Ingress`.
+
+5.  The program will ask for a cluster domain. Default is the one
+created in [the Cluster creation
+process](/docs/cloudbees-jenkins-x-distribution/boot/) such as
+`192.168.1.100.nip.io`.
+
+6.  The program will ask for an `URLTemplate`. Press `Enter` to
+use the default value.
+
+7.  The program will verify your answers to the previous questions in
+summary and prompt you to approve the Vault creation (default is
+`Yes`).
+
 ## Using Vault with boot
```

Comment:
@jha-cloudbees I think we can remove this section `Using Vault with boot`. It is already covered by your docs.

Reply:
removed the section completely